### PR TITLE
initial filter set

### PR DESCRIPTION
**Pull request by JauniusZ**
Added a function to set filter.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -65,3 +65,18 @@ function updateUncheckedTodoCount() {
   const todosLeftElement = document.getElementById("todos-left");
   todosLeftElement.textContent = `Todos left unchecked: ${uncheckedTodos}`;
 }
+
+function setFilter(filterValue) {
+  const filterRadioButton = document.getElementById(
```
Comment (arturasmckwcz) [inline script.js]: NIT: Why do you need so complicated expression for id? You could easily select by value:
```
const filters = document.getElementById("filters");
...
function setFilter(filterValue) {
const filterRadioButton = filters.querySelector('input[value=${filterValue}]')
...
```

File: script.js [hunk 2]
```diff
@@ -65,3 +65,18 @@ function updateUncheckedTodoCount() {
...
+    `option${filterValue === "all" ? 1 : filterValue === "completed" ? 3 : 2}`
+  );
+  if (filterRadioButton) {
+    filterRadioButton.checked = true;
+    state.filter = filterValue;
```
Comment (arturasmckwcz) [inline script.js]: This assignment here is kind of strange. I'd consider setting the state first and then call something to render reflecting that change, right?

File: script.js [hunk 3]
```diff
@@ -65,3 +65,18 @@ function updateUncheckedTodoCount() {
...
+    renderTodos();
+  } else {
+    console.error("Invalid filter value provided.");
+  }
+}
+
+setFilter("all");
```
Comment (arturasmckwcz) [inline script.js]: Apparently you have called the function once. I believe you won't need to call it more than once later on as well. Unless you plan to save the filter state in between browser sessions, so you will need to set the filter from saved state. But I think it isn't needed as user might be happy with initial value 'all' each time the app gets reloaded, right?
What am I heading to? You do not need javascript here at all just set `checked` radio button in html and the filter state is already initially set to 'all'.

**Comment by JauniusZ** [comment]
Set checked radio button in index.html and got rid of the setFilter function in script.js. Is this what you meant?

**Review by arturasmckwcz — APPROVED** [review]
great, thank you

**Pull request merged**